.pre-commit-config.yaml

```diff
@@ -49,6 +49,7 @@ repos:
     hooks:
     - id: ruff
       args: [--fix]
+      exclude: 'examples'
```

Comment:
1. Is this exclusion of the examples folder going to be a general rule or is it because you would rather not have to lint the files in the present folder right now? 

2. Please keep exceptions and exclusions to the `pyproject.toml` file under the `[tool.ruff]` header.